src/consensus/cert.rs

```diff
@@ -574,9 +576,9 @@ mod tests {
                 stake: 1,
                 pubkey: sks.last().unwrap().to_pk(),
                 voting_pubkey: voting_sks.last().unwrap().to_pk(),
-                all2all_address: String::new(),
-                disseminator_address: String::new(),
-                repair_address: String::new(),
+                all2all_address: sockaddr,
```

Comment:
maybe call `dontcare_sockaddr()` here directly, instead of using `sockaddr` variable